Scripts/lombiq-disableemptyforminputs.js

```diff
@@ -17,6 +17,11 @@
 
                 return true; // To ensure that the form still submits.
             });
+
+            // Un-disable form fields when page loads, in case they click back after submission.
+            element.find(":input").filter(function () {
+                return !this.value && !$(this).is("[dontdisableifempty]");

```

Comment:
You can move the second check into the selector:
```suggestion
            element.find(":input:not([dontdisableifempty])").filter(function () {
                return !this.value;
```

Reply:
Nice catch! Thanks, Oliver!